### PR TITLE
Tc 3982

### PR DESCRIPTION
**Pull request by sasha-3C**
Redesigned QTD UI display for activity feed.

### CODE REVIEW

File: css/scss/apps/thirdchannel/_activities.scss [hunk 1]
```diff
@@ -149,14 +147,33 @@ $padding: 1.5em;
...
+
+    .sales-button {
+      padding-left: 1em;
+
+      .change-icon {
+        font-size: 1em;
+      }
+    }
+
+    .pure-g {
```
Comment (spullen) [inline css/scss/apps/thirdchannel/_activities.scss]: You may want to just add an extra class just so it doesn't conflict with the pure-g library.

File: css/scss/apps/thirdchannel/_activities.scss [hunk 2]
```diff
@@ -14,11 +14,11 @@ $padding: 1.5em;
   @include clearfix();
   padding: $padding $padding 0 $padding;
   vertical-align: middle;
-  border-bottom: 1px solid $border-color;
+  //border-bottom: 1px solid $border-color;
```
Comment (spullen) [inline css/scss/apps/thirdchannel/_activities.scss]: If we're getting rid of this you can just delete it.

File: css/scss/apps/thirdchannel/_activities.scss (hunk 1 above)
Comment (sasha-3C) [inline css/scss/apps/thirdchannel/_activities.scss]: Now that I'm looking at it, don't think I even need this class at all. I'll delete the line

Comment (spullen) [inline css/scss/apps/thirdchannel/_activities.scss]: There's an element with `col-2-3` in the container that you removed the `pure-g` class from, are you sure you don't need it?

Comment (sasha-3C) [inline css/scss/apps/thirdchannel/_activities.scss]: I tried deleting it on dev and it didn't seem to break anything. Deleted the col-2-3 thing too and that didn't affect anything either

**Comment by spullen** [comment]
Looks fine to me

**Pull request merged**